save-points/6-Deployment-docker/ConferencePlanner/docker-compose.override.yml

```diff
@@ -1,6 +1,10 @@
-version: '3.4'
+version: '3.3'

```

Comment:
Is there a reason to go from 3.4 to 3.3 here?

Reply:
VSTS Build Agent does not support 3.4 yet. No major difference.

Reply:
Cool, makes sense.